README.md

```diff
@@ -472,7 +472,7 @@ const mp = new Minipass({ encoding: 'utf8' })
 // some source of some data
 let i = 5
 const inter = setInterval(() => {
-  if (i --> 0)
+  if (i-- > 0)
```

Comment:
It's so cute, though!  Like a little arrow ➡️

Reply:
LOL, sure was. 😄